lib/decanter.rb

```diff
@@ -5,14 +5,20 @@ module Decanter
   class << self
```

Comment:
The tests are failing on master branch right now. Changes to this file fix those failing specs.